pkg/library/home/persistentHome.go

```diff
@@ -27,6 +27,24 @@ import (
 	"github.com/devfile/devworkspace-operator/pkg/constants"
 )
 
+const initScript = `(echo "Checking for stow command"
+STOW_COMPLETE=/home/user/.stow_completed
+if command -v stow &> /dev/null; then
+  if  [ ! -f $STOW_COMPLETE ]; then
+    echo "Running stow command"
+    stow . -t /home/user/ -d /home/tooling/ --no-folding -v 2 > /home/user/.stow.log 2>&1
+    cp -n /home/tooling/.viminfo /home/user/.viminfo
+    cp -n /home/tooling/.bashrc /home/user/.bashrc
+    cp -n /home/tooling/.bash_profile /home/user/.bash_profile
+    touch $STOW_COMPLETE
```

Comment:
It'd be nice to log here that the init script completed here, but that can be added in a future PR. We could also check whether stow completed successfully and report that here.